release/preview/fedora/docker/Dockerfile

```diff
@@ -22,21 +22,48 @@ ENV DOTNET_SYSTEM_GLOBALIZATION_INVARIANT=false \
 # Install dependencies and clean up
 RUN curl -sSL ${PS_PACKAGE_URL} -o /tmp/powershell.rpm \
     && dnf install -y /tmp/powershell.rpm \
+    # remove powershell package
+    && rm /tmp/powershell.rpm \
     && dnf install -y \
     # less is needed for help
-    less \
+      less \
     # Needed to run localdef
-        glibc-locale-source \
+      glibc-locale-source \
     # Invoke-WebRequest doesn't work correctly without this
-    compat-openssl10 \
-    ca-certificates \
-    gssntlmssp \
+      compat-openssl10 \
+      ca-certificates \
+      gssntlmssp \
+    && ln -sf /opt/microsoft/powershell/7-preview/pwsh /usr/bin/pwsh \
+    # For whatever reason FEDORA-2019-27e7b92407 has to be patched manually
+    # to do this, upgrade libmodulemd1, if it is installed
+    && pwsh \
+        -NoLogo \
+        -NoProfile \
+        -Command " \
+          \$module = (dnf list installed libmodulemd1 | Select-String -SimpleMatch libmodulemd1);  \
+          if(\$module) { \
+             dnf upgrade -y libmodulemd1 \
+          }" \
+    # This installs most security advisories
     && dnf upgrade-minimal -y --security \
+    # query and install any remaining security advisories
+    && pwsh \
+        -NoLogo \
+        -NoProfile \
+        -Command " \
+          (dnf updateinfo list -q --security) |  \
+              Foreach-Object { \
+                  \$advisory=(\$_ -split ' ')[0]; \
+                  Write-Host '******* Patching *********'; \
+                  Write-Host \$advisory; \
+                  Write-Host '************************'; \
+                  dnf upgrade -y --advisory=\$advisory \
+              }" \
+    && echo "verifying all security advisories are installed..." \
```

Comment:
What happens if some are not installed?

Reply:
nothing... just prints information.

Reply:
I'd rather this be a test:  https://github.com/PowerShell/PowerShell-Docker/issues/323